src/filter/bgsubtract0r/bgsubtract0r.c

```diff
@@ -86,7 +86,7 @@ f0r_instance_t f0r_construct(unsigned int width, unsigned int height)
   inst->blur = 0;
   inst->threshold = 26;
   inst->reference = NULL;
-  inst->mask = malloc(width*height);
+  inst->mask = malloc(sizeof(uint32_t)*width*height);
```

Comment:
This would be correct according to the way memset() is called on line 204. However, looking at how mask is used, it only ever stores bytes in the amount of width*height. You did find a problem, but the problem is the memset() call and not here. memset should be called with `len` instead of `blen`.